src/qdrant.rs

```diff
@@ -7480,4 +7516,42 @@ where
 builder_type_conversions!(CreateCollection, CreateCollectionBuilder);
 builder_type_conversions!(VectorParams, VectorParamsBuilder);
 builder_type_conversions!(HnswConfigDiff, HnswConfigDiffBuilder);
+builder_type_conversions!(ScalarQuantization, ScalarQuantizationBuilder);
+builder_type_conversions!(ProductQuantization, ProductQuantizationBuilder);
+builder_type_conversions!(BinaryQuantization, BinaryQuantizationBuilder);
+builder_type_conversions!(OptimizersConfigDiff, OptimizersConfigDiffBuilder);
+builder_type_conversions!(WalConfigDiff, WalConfigDiffBuilder);
+
+impl VectorParamsBuilder {
+    pub fn new(size: u64, distance: Distance) -> Self {
+        let mut builder = Self::create_empty();
+        builder.size = Some(size);
+        builder.distance = Some(distance.into());
+        builder
+    }
+}
+
+impl ScalarQuantizationBuilder {
+    pub fn new(r#type: QuantizationType) -> Self {
+        let mut builder = Self::create_empty();
+        builder.r#type = Some(r#type.into());
+        builder
+    }
+}
+
+impl ProductQuantizationBuilder {
+    pub fn new(compression: i32) -> Self {
+        let mut builder = Self::create_empty();
+        builder.compression = Some(compression);
+        builder
+    }
+}
+
+impl BinaryQuantizationBuilder {
+    pub fn new(always_ram: bool) -> Self {
+        let mut builder = Self::create_empty();
+        builder.always_ram = Some(Some(always_ram));
+        builder
+    }
+}
```

Comment:
Those functions get appended on code generation from [here](https://github.com/qdrant/rust-client/blob/8294725bcba81211f802105113da2e40b88836e0/tests/protos_append/builder_ext.rs)
I thought it would be cleaner to have those separate impls in a different file than the macro file.

Reply:
What I meant is, maybe they'll still work without copying them to the generated `qdrant.rs` file?